src/app/features/auth/components/login-form/login-form.component.ts

```diff
@@ -0,0 +1,65 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, Output } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { AbstractControl, FormGroup, NonNullableFormBuilder, ReactiveFormsModule, Validators } from "@angular/forms";
+import { Credentials } from "../../../../core/auth/types/credentials";
+
+@Component({
+  selector: 'login-form',
+  standalone: true,
+  imports: [CommonModule, ReactiveFormsModule],
+  templateUrl: './login-form.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class LoginFormComponent {
+
+  @Input() public submitted: boolean | null;
+  @Output() public readonly submitEmitter = new EventEmitter<{ credentials: Credentials; isRemember: boolean }>();
```

Comment:
1. Do not add angular-specifi suffixes to names. Why the `Emitter` suffix is needed here? Is it unclear without it?
2. I see you're making the same type more than once already. You probably need to declare a separate type.